compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/EnumClassLowering.kt

```diff
@@ -156,8 +153,14 @@ private class EnumClassLowering(val context: JvmBackendContext) : ClassLoweringP
                 declaration.body = context.createJvmIrBuilder(declaration.symbol).run {
                     irExprBody(
                         when (body.kind) {
-                            IrSyntheticBodyKind.ENUM_VALUES ->
-                                irArray(valuesField.type) { addSpread(irGetField(null, valuesField)) }
+                            IrSyntheticBodyKind.ENUM_VALUES -> {
+                                val cloneFun = context.irBuiltIns.arrayClass.functions.first {
```

Comment:
Let's use `single` instead of `first`, just in case. :)

Reply:
Sure! Done.